docs/writing-data.md

```diff
@@ -113,6 +113,30 @@ This will result in the following pieces of Kafka record metadata being in each
 - `kafka.partition` = the partition of the Kafka record
 - `kafka.timestamp` = the timestamp of the Kafka record
 
+### Including Kafka headers
+
+Each Kafka record passed to the MarkLogic connector also has headers that may contain useful information which can be
+included in the metadata written with documents. This includes the headers that are included in Kafka records by
+default as well as any custom headers. Kafka headers can be included in each document by configuring the following
+property:
+
+- `ml.dmsdk.includeKafkaHeaders` = `true` to include Kafka headers
+
+When the headers are added to the document metadata, they are simply given the same name as the key for the header.
+However, you may also specify a prefix that will be prepended to each header key. To set that prefix, use the following
+property:
+
+- `ml.dmsdk.includeKafkaHeaders.prefix` = `<prefix>` to be prepended to header keys in the metadata.
+
+This will result in the following pieces of Kafka record metadata being in each associated document as MarkLogic
+[document metadata values](https://docs.marklogic.com/guide/java/document-operations#id_83108):
+
+- `kafka.topic` = the Kafka topic associated with the record
+- `kafka.key` = the key of the Kafka record (may be null)
```

Comment:
These are already included by setting `ml.dmsdk.includeKafkaMetadata=true` - does setting `includeKafkaHeaders=true` include them a second time? Or are these bits not included as part of Kafka headers?